.travis/playbook.yml

```diff
@@ -0,0 +1,29 @@
+---
+- hosts: all
+  pre_tasks:
+    - name: Load DB variables
+      include_vars: '{{ pulp_db_type }}.yml'
+  vars:
+    pulp_default_admin_password: admin
+    pulp_secret_key: secret
+    pulp_source_dir: '/home/travis/build/pulp/pulpcore/'
+    pulp_plugin_source_dir: "/home/travis/build/pulp/pulpcore-plugin"
+    pulp_install_plugins:
+      pulp-file:
```

Comment:
This needs to be updated.

Reply:
Thanks! I updated this and the following line.